File: Module/PlayWays/src/main/java-songmanage/com/module/playways/songmanager/view/IGrabWishManageView.java
package com.module.playways.songmanager.view;

import com.module.playways.songmanager.model.GrabWishSongModel;

import java.util.List;

public interface IGrabWishManageView {
    void addGrabWishSongModels(boolean clear, long newOffset, List<GrabWishSongModel> grabWishSongModels);

    void deleteWishSong(GrabWishSongModel grabWishSongModel);
}

File: Module/PlayWays/src/main/java-songmanage/com/module/playways/songmanager/view/IGrabWishManageView.kt
package com.module.playways.songmanager.view

import com.module.playways.songmanager.model.GrabWishSongModel

interface IGrabWishManageView {
    fun addGrabWishSongModels(clear: Boolean, newOffset: Long, grabWishSongModels: List<GrabWishSongModel>?)

    fun deleteWishSong(grabWishSongModel: GrabWishSongModel)
}
